README.md

```diff
@@ -1,23 +1,22 @@
 # Temporal Python SDK Samples
 
-This is the set of Python samples for the [Python SDK](https://github.com/temporalio/sdk-python).
+This is a collection of samples showing how to use the [Python SDK](https://github.com/temporalio/sdk-python).
 
 ## Usage
 
 Prerequisites:
 
-* Python >= 3.9
```

Comment:
Can we leave the Python version constraints?

Reply:
We don't want users thinking they have to install a special Python version. uv users should install python using

```
uv python install 3.9
```

Reply:
Happy to clarify somewhere that they should run that command.

Reply:
> We don't want users thinking they have to install a special Python version. uv users should install python using

Right, "Python >= 3.9" just means "the software in this repository needs at least Python 3.9", it doesn't say how to install it or that it has to be any specific version. Our SDK and samples repo READMEs need to clarify which versions of Python they work with.

Reply:
How about this?

Reply:
LGTM